Lombiq.UIKit/wwwroot/js/slick-carousel.js

```diff
@@ -1,13 +1,12 @@
 jQuery(($) => {
     document.querySelectorAll('.slickCarousel').forEach((containerElement) => {
         const carousel = containerElement.querySelector('.slickCarousel__carousel');
-
         const findItems = () => Array.from(carousel.querySelectorAll('.slickCarousel__item'));
 
         let carouselSettings = {
             autoplay: true,
             mobileFirst: true,
-            appendArrows: '.slickCarousel__arrows',
+            appendArrows: '#' + containerElement.id + ' .slickCarousel__arrows',
```

Comment:
This makes the ID a hidden dependency. According [to the docs](https://kenwheeler.github.io/slick/#:~:text=appendArrows) the value of `appendArrows` can be `Element` as well, so can you try this instead?
```suggestion
            appendArrows: containerElement.querySelector('.slickCarousel__arrows'),
```

------------

This shouldn't be necessary and it's definitely a bug in the library. But [after recent findings](https://teams.microsoft.com/l/message/19:525a1a3656c845ae92c5e2c0d3617cc7@thread.skype/1750696162143?tenantId=959ede5a-38e9-44e6-adc8-85553beb8220&groupId=d6a5d775-6fb9-41dc-a7bb-481d4f884046&parentMessageId=1750695781420&teamName=Lombiq%20Developers&channelName=General&createdTime=1750696162143) I won't ask you to make a bug report in their repo as it seems pointless.

Reply:
That's true, this is really a better solution!